src/pixelator/pna/amplicon/build_amplicon.py

```diff
@@ -488,6 +489,31 @@ def _scan_reverse_read(self, read: SequenceRecord) -> AmpliconRegionSlices:
 
         return region_slices
 
+    @staticmethod
+    def _get_region_sequence(read, region_slice, is_reversed=False, as_bytearray=False):
```

Comment:
Might be good to add return type hints here:

```suggestion
    @staticmethod
    def _get_region_sequence(read, region_slice, is_reversed=False, as_bytearray=False) -> tuple[bytes | None, bytes | None]:
```